.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>Gadgetbridge</name>
+	<comment>Project Gadgetbridge created by Buildship.</comment>
+	<projects>
+	</projects>
+	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.buildship.core.gradleprojectbuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.buildship.core.gradleprojectnature</nature>
+	</natures>
+</projectDescription>
```

Comment:
Is there any specific reason why you use Buildship instead of Android Studio and do we need to have these files in the repository?

Reply:
I think these should be added to .gitignore and removed from the PR. Especially if they're autogenerated.